### PR TITLE
Add demos

### PR DESCRIPTION
**Pull request by Tobi406**
Add demos (or demo buttons) and errorhandling to Editor, Verbose and Tree pages.

## To-Do
- Add demo files (or content) for
  - [x] Verbose
  - [x] Tree
- [x] Add errorhandling to Tree
- [x] Find out why Editor button looks different and adjust the other ones (or other way around)

## How it looks like
image
image

### CODE REVIEW

**Comment by Tobi406** [comment]
Also do change tabs

File: src/views/Verbose.vue [hunk 1]
```diff
@@ -137,14 +145,18 @@
...
 </script>
 
 <style lang="scss">
   main.verbose {
     display: flex;
     overflow-y: hidden;
+		
+		.demo-button {
+			font-size: 1.5em;
+		}
```
Comment (Tobi406) [inline src/views/Verbose.vue:159]: Not sure if I shoudl apply these rules to `common.scss` or something like that, suggestions?

Comment (Turbotailz) [reply src/views/Verbose.vue]: I think once something is used in multiple components, it should be added to `common.scss`, but it's not hugely important.

**Comment by Turbotailz** [comment]
This is really awesome, thank you!

My only suggestion is the buttons look a bit large, I think they would look better with normal font-size (`1rem`)

File: src/views/Editor.vue [hunk 2]
```diff
@@ -224,6 +220,7 @@ main.editor {
         display: flex;
         flex-direction: column;
         position: relative;
+        border-right: 1px solid rgba(255, 255, 255, 0.2);
```
Comment (Tobi406) [inline src/views/Editor.vue:223]: That shouldn't be there, was testing out something else. Will probably make another PR for that and other CSS changes.
